src/inversion_scripts/utils.py

```diff
@@ -293,6 +294,8 @@ def filter_tropomi(tropomi_data, xlim, ylim, startdate, enddate):
         & (tropomi_data["time"] <= enddate)
         & (tropomi_data["qa_value"] >= 0.5)
         & (tropomi_data["longitude_bounds"].ptp(axis=2) < 100)
+        & (tropomi_data["surface_classification"] != 1)
+        & (tropomi_data["latitude"] > -60)
```

Comment:
@djvaron back in that long email thread about new boundary conditions, we decided to drop data over the water and south of 60S. This means it will be dropped in inversions as well (to be consistent with boundary conditions). Is that okay?